### ISSUE TITLE
Improve highlighting capabilities

### ISSUE THREAD

**Issue by Spriithy** [reasonml-editor/reasonml-idea-plugin#13]
I think your plugin should be able to highlight numeric constants as in

```ocaml
let x = 10
```

and type constants as in

```ocaml
type tree 'a =
    | Leaf of 'a
    | Tree of tree 'a * tree 'a
```

Do you think you could do that ?

**Comment by giraud** [comment]
Thanks for the suggestion, this is what version 0.24 can do:

![highlight](https://github.com/reasonml-editor/reasonml-idea-plugin/blob/master/docs/issue_highlight.png)

of course, reason syntax only

**Comment by Spriithy** [comment]
Would it be OK for you ton include syntax changes for OCaml in general ?

**Comment by giraud** [comment]
fix in 0.48
